src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/EstimatedAsset2.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd;
+
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.RequiredArgsConstructor;
+
+import java.util.Arrays;
+
+@RequiredArgsConstructor
+public enum EstimatedAsset2 {
```

Comment:
This name is confusing. If there really wasn't something more appropriate then a Javadoc would help explain the "2"

Reply:
I agree in parts, I am not a fan of javadocs as they are often are used to masquerade  bad design. However the intention for the change can be indeed improved with better naming so I will change this to V2 instead where I believe it clearly states then, its intention to be a second version, up to date list of values.